docs/controller-skills/sequencing.md

```diff
@@ -8,12 +8,88 @@ title: Sequencing
 
 It's all well and good maintaining separation standards in your airspace, but if you handoff 2 aircraft to ML_APP overhead LIZZI at the same speed, 1000ft apart, you're not going to make any friends. By default, aircraft must be sequenced **2 minutes** apart at the **Feeder Fix** (generally the first waypoint of the STAR).  
 
-It's important to remember that aircraft aren't just arriving from your sector, they can be coming in from all directions at similar times. **MAESTRO** is a tool used to help aid with sequencing (what time do YOU need to get the aircraft to the feeder fix to avoid TCU congestion). You can find more info [here](../../controller-skills/maestro)
+It's important to remember that aircraft aren't just arriving from your sector, they can be coming in from all directions at similar times. **MAESTRO** is a tool used to help aid with sequencing (what time do YOU need to get the aircraft to the feeder fix to avoid TCU congestion). You can find more info [here](../maestro).
 
 In real life, controllers have the luxury of putting a fair bit of the onus of forming the sequence on the pilots, by issuing instructions like *"Adjust speed to cross RIVET at time 52 at 250kts in to published speeds"*. On VATSIM, every pilot may be using a different clock, potentially even different weather, and the pilot may simply not be proficient enough to meet a FF time. As such, all sequencing is best left in the hands of you, the controller.
 
 There are multiple tools a controller can use to create and maintain a sequence.
 
+## Arrivals List
+The Arrivals List window is a sequencing management tool that can be used side-by-side with, or in-lieu of, [Maestro](../maestro).
+
+It is a **global** window that can be viewed and manipulated by Tower, TCU, and Enroute Controllers.
+
+The responsibility for managing the window falls on the:
+
+- **Flow** Controller
+    - This is delegated to the **Approach** Controller when Flow is Offline
```

Comment:
Ideally the TMA guy would do it, but we don't really teach that to S3s. Maybe we could say 'Approach or surrounding Enroute controller'

Reply:
Yeah my thinking is, it's fine if people want to not use it due to lack of training, very quiet airport, etc. But if they want to use it, the info is there in the SOPs

In my opinion, it's not of much value to be managed by a surrounding enroute controller. At YMML and YSSY, there are multiple surrounding enroute controllers. And even at YBBN, YPAD, and YPPH where there's only 1, the Enroute controller doesn't really have a picture of how they conflict in the sequence, they only care about separating aircraft through the same FF.

So yeah, my thoughts are, the actions affect, and are for the benefit of, the approach/flow controller. Therefore, it should be managed by them